Automattic/jetpack
# Contact Form: Sidebar Revamp

Fixes #26945

#### Changes proposed in this Pull Request:
* Include "Manage Responses" Section
* Split the "Forms Settings" section into "Submission Settings" and "Email Connection"

Before:
![image](https://user-images.githubusercontent.com/7811225/197026928-4ae01cf7-4a76-4cc6-8daa-2f47f89e5195.png)

After:
![image](https://user-images.githubusercontent.com/7811225/197028949-d0406889-5691-4c74-b556-6e8c8d018a59.png)

#### Other information:


#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Checkout this branch and create a Page/Post
* Add a Form block to the page and check if the block's Sidebar matches the description above
